openpype/hosts/photoshop/plugins/publish/extract_review.py

```diff
@@ -120,6 +136,35 @@ def process(self, instance):
 
         self.log.info(f"Extracted {instance} to {staging_dir}")
 
+    def _check_and_resize(self, processed_img_names, source_files_pattern,
+                          staging_dir):
+        """Check if saved image could be used in ffmpeg.
+
+        Ffmpeg has max size 16384x16384. Saved image(s) must be resized to be
+        used as a source for thumbnail or review mov.
+        """
+        # 16384x16384 actually didn't work because int overflow
+        max_ffmpeg_size = 16000
```

Comment:
Maybe let's use 8K (8192).

Reply:
changed to 8k. It actually worked with 16K on our testing PC, but I guess 8K is plenty for review purposes.